scripts/kconfig/kconfigfunctions.py

```diff
@@ -138,6 +138,25 @@ def dt_chosen_label(kconf, _, chosen):
     return node.props["label"].val
 
 
+def dt_chosen_enabled(kconf, _, chosen):
+    """
+    This function takes a 'chosen' property and returns "y" if we find an
+    "enabled" node pointed to by the 'chosen' property in the EDT otherwise
+    we return "n"
```

Comment:
Could change to

    Returns "y" if /chosen contains a property named 'chosen' that points to
    an enabled node, and "n" otherwise